feature/feature.xml

```diff
@@ -34,7 +34,7 @@ THE SOFTWARE.
    </license>
 
    <url>
-      <update label="StartExplorer Update Site" url="https://fabioz.github.com/startexplorer/update/"/>
+      <update label="StartExplorer Update Site" url="http://basti1302.github.io/startexplorer/update/"/>
```

Comment:
By no means can that be the correct fix. I maintained startexplorer roughly until 2015 or so. For some reason, the old github pages URLs under my user name still seem to work, probably because of the permanent redirect from the basti1302/startexplorer repository URL (and associated github pages) to @fabioz account.

I guess the correct URL would be https://fabioz.github.io/startexplorer/update/ (not the `.io` versus `.com`). Please do not use my Github handle `basti1302` here since I am no longer involved in the project.